src/main/webapp/WEB-INF/tags/fileUploader.tag

```diff
@@ -0,0 +1,40 @@
+<%@ tag language="java" pageEncoding="UTF-8"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@attribute name="attachmentsTarget" type="org.mamute.model.Post" required="false"%>
+
+<c:if test="${env.get('feature.inhouse.upload')}">
+    <div class="uploader">
+        <label>${t['question.attachments']} - <a class="add-file" href="#">${t['question.attachments.newfile']}</a></label>
```

Comment:
Well all the examples and tag attributes are related to form, so I guess it's not semantic to use it anywhere we want :/
Since the text you want is `Attachments`, it sounds like a title